driver-mapping/src/main/java/com/datastax/driver/mapping/Mapper.java

```diff
@@ -127,7 +128,38 @@ public MappingManager getManager() {
      * @return a query that saves {@code entity} (based on it's defined mapping).
      */
     public Statement saveQuery(T entity) {
```

Comment:
To avoid code duplication, you can have this call `saveQuery(entity, defaultSaveOptions)`, provided that the case where the defaults are null is handled correctly (which I think is the case).